static/app/views/explore/contexts/spanTagsContext.tsx

```diff
@@ -12,6 +12,47 @@ import usePrevious from 'sentry/utils/usePrevious';
 import {SpanIndexedField} from 'sentry/views/insights/types';
 import {useSpanFieldCustomTags} from 'sentry/views/performance/utils/useSpanFieldSupportedTags';
 
+export const SentryStringTags = [
```

Comment:
Nit, as this becomes shared across more places, we should consider moving it to a constants file at some point.

Reply:
Moved it to explore/constants.tsx